dashboard/app/models/user.rb

```diff
@@ -126,6 +130,8 @@ class User < ApplicationRecord
     gender_student_input
     gender_teacher_input
     gender_third_party_input
+    cpa_compliance
+    cpa_compliance_date
```

Comment:
Should we make these names generic so that we might be able to extend their use to future parent permission scenarios outside of CPA? e.g. making it something like `parent_permission_granted` and `parent_permission_date`?

Reply:
Good idea. I have decided to change the terminology from CPA to "Child Account Policy". That way, as other child protections come into play, this naming will still be valid as we tweak it. For example, if protections should be extended to U14 instead of U13.